### ISSUE TITLE
Change password

### ISSUE THREAD

**Issue by Siegi96** [adw0rd/instagrapi#834]
Hey, is there an endpoint for manually changing password(without challenge)? Couldn't find something like that.

**Comment by adw0rd** [comment]
@Siegi96 OK, I'm waiting for your PR, right?

**Comment by SaeidB** [comment]
> Hey, is there an endpoint for manually changing password(without challenge)? Couldn't find something like that.

yes , there is in instagram app , its possible to add it to  instagrapi

**Comment by subzeroid** [comment]
This is implemented in current releases via `Client.change_password(old_password, new_password)`.

Docs: https://subzeroid.github.io/instagrapi/usage-guide/account

Closing this old feature request as completed.
